contribute/process/commit_template

```diff
@@ -0,0 +1,16 @@
+# <50 character summary of the change
+
+# 1. Why is this change necessary ?
+
+
+# 2. How does this change address the issue ?
+
+# fix: #1234
```

Comment:
This sample text is wrong.
A commit can fix only a single #issue. The issue in turn can be linked to other issues.
Give a sample answer that is detailed.

Reply:
This is something I picked up from Openstack.
This isn't part of "How does this change address the issue"